raphtory-api/src/core/entities/properties/meta.rs

```diff
@@ -104,6 +104,20 @@ impl Meta {
         }
     }
 
+    pub fn new_for_graph_props() -> Self {
+        let meta_layer = DictMapper::new_layer_mapper();
+        let meta_node_type = DictMapper::default();
+        meta_node_type.get_or_create_id("_default");
```

Comment:
no reason to add this?

Reply:
Removed, I suspect we don't need it for `new_for_edges` either.